spec/support/helpers/dependency_helper.rb

```diff
@@ -10,7 +10,11 @@ def rails_present?
   end
 
   def active_record_present?
-    dependency_present? 'active_record'
+    dependency_present? 'activerecord'
```

Comment:
I see.. Yeah it should use the gem name now and not the file to require.

However, I also see we don't use this check anywhere. That's why it hasn't raised a problem anywhere. best to just remove it?